neoassistant/commands.py

```diff
@@ -241,10 +241,22 @@ def execute(self, assistant: Assistant, args):
 
 class ShowBirthdaysCommand(Command):
     def __init__(self):
-        super().__init__("show-birthdays", "Show all birthdays per next 7 days.")
+        super().__init__(
+            "show-birthdays", 
+            "Show all birthdays per the next specified number of days," + 
+            " if days are not specified show per next 7 days."
+            )
 
-    def execute(self, assistant: Assistant, _):
-        return assistant.contact_book.get_birthdays_per_week()
+    def execute(self, assistant: Assistant, args):
+        days_delta = 7
+        if args:
+            try:
+                days_delta = int(args[0])
+                if days_delta > 365:
+                    return "The maximum value for days_delta is 365."
```

Comment:
Please raise an error instead of returning a string: `raise InvalidCommandError(self.name, "The maximum value for days_delta is 365.")`